app/hotels/src/navigation/HotelsNavigationOptions.js

```diff
@@ -68,6 +68,7 @@ const hotelsNavigationOptions = ({
             }
             date={checkin}
             onDateChange={setCheckinDate}
+            minDate={DateUtils.getToday()}
```

Comment:
Don't know if that makes a huge difference, but here you're creating 3 instances of `Date` by calling `DateUtils.getToday()`. Since `addDays` should be immutable, you could probably get away with creating just one Date instance and then call addDays to it below? WDYT?

Reply:
> Don't know if that makes a huge difference

It should not

Reply:
What I mean is: not in this one instance, but if we did this everywhere, then we would be consuming more memory than necessary.

Reply:
🚀 

I think we saved memory consumption of all users of kiwi.com app 🎉